python/ray/_raylet.pyx

```diff
@@ -872,6 +879,7 @@ cdef class CoreWorker:
                     function_descriptor,
                     args,
                     int num_return_vals,
+                    c_bool is_direct_call,
```

Comment:
Why not make this a global parameter? Do you want to mix direct and non-direct calls?

Reply:
Yeah, this allows libraries etc to be incrementally ported without flipping everything. It's also useful for testing scripts.